components/datetime/src/neo_marker.rs

```diff
@@ -1057,7 +1057,7 @@ macro_rules! impl_date_marker {
         input_day_of_year = $day_of_year_yesno:ident,
         input_any_calendar_kind = $any_calendar_kind_yesno:ident,
     ) => {
-        #[doc = concat!("Marker for ", $description, ": ", $expectation)]
+        #[doc = concat!("**“", $expectation, "**” ⇒ ", $description)]
```

Comment:
The quotation marks are because it is a human-readable output string as opposed to developer docs.

The boldface is to make it easier to read in a list as shown above.

The arrow was just some character I thought looked nice. Happy for suggestions on another character to use in that location.

Reply:
I do like the boldface, and I do think the quotes are contextually needed here. We could just use a colon

Reply:
I assume @robertbastian's comment was non-blocking. These are docs and we can continue to improve.